index.js

```diff
@@ -117,8 +118,14 @@ StackUtils.prototype.capture = function (limit, fn) {
 	}
 	var prepBefore = Error.prepareStackTrace;
 	var limitBefore = Error.stackTraceLimit;
+	var wrapCallSite = this._wrapCallSite;
 
 	Error.prepareStackTrace = function (obj, site) {
+		if (typeof wrapCallSite === 'function') {
```

Comment:
just do `if wrapCallSite`. If they pass a non-function, it should produce a TypeError